src/@types/muiTreasury.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@mui-treasury/components/chatMsg/ChatMsg';
```

Comment:
The `@types` folder is used to create declaration files for libraries that don't export their own declaration files (aka non-TypeScript packages) or have their typing on Definately Typed (aka `yarn add -D @types/<package>`).

You could add all the typings for the package if you really wanted, I have done it in the past for smaller packages, but this format will satisfy the type error, though you won't have any typings for the imported module.